### PR TITLE
Renamed to DataCiteMetadataDto

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
+     * @param dataCiteMetadataDto A DynamoDB record.
      * @throws JAXBException If the XML initialisation fails.
      */
-    public Transformer(DynamoRecordDto dynamoRecordDto) throws JAXBException {
+    public Transformer(DataCiteMetadataDto dataCiteMetadataDto) throws JAXBException {
         marshaller = jaxbContext.createMarshaller();
         marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, Boolean.TRUE);
         this.resource = new Resource();
-        fromDynamoRecord(dynamoRecordDto);
+        dataCiteMes(dataCiteMetadataDto);
```
Comment (norrs) [inline datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java]: dataCiteMes?

**Review by norrs — CHANGES_REQUESTED** [review]

File: datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java (hunk 1 above)
Comment (janbj) [reply datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java]: Hm, fixed

File: datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java [hunk 2]
```diff
@@ -24,14 +24,14 @@
 
     /**
      * Transforms a DynamoDB record to a Datacite Record.
-     * @param dynamoRecordDto A DynamoDB record.
+     * @param dataCiteMetadataDto A DynamoDB record.
```
Comment (norrs) [inline datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java]: DynamoDB probably should be replaced with DataCiteMetadataDto to hide storage impl.

Comment (norrs) [reply datacite-transform/src/main/java/no/unit/nva/transformer/Transformer.java]: So ensure the javadoc is correct.

**Review by norrs — CHANGES_REQUESTED** [review]

**Review by brinxmat — APPROVED** [review]

**Review by norrs — APPROVED** [review]

**Comment by norrs** [comment]
Since @janbj is AFK this week(?), I'm merging this for him @brinxmat

**Pull request merged**